openstack/networking/v2/extensions/rbacpolicies/requests.go

```diff
@@ -0,0 +1,51 @@
+package rbacpolicies
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// PolicyAction maps to Action for the RBAC policy.
+// Which allows access_as_external or access_as_shared.
+type PolicyAction string
+
+const (
+	//ActionAccessExternal returns Action for the RBAC policy as access_as_external.
+	ActionAccessExternal PolicyAction = "access_as_external"
+
+	//ActionAccessShared returns Action for the RBAC policy as access_as_shared.
```

Comment:
doc nit: there should be a space between `//` and `ActionAccessShared`

Reply:
Done